Mathlib/CategoryTheory/Bicategory/Extension.lean

```diff
@@ -129,8 +129,9 @@ def whiskerHom (i : s ⟶ t) {x : B} (h : c ⟶ x) :
   StructuredArrow.homMk (i.right ▷ h) <| by
     rw [← cancel_mono (α_ _ _ _).inv]
     calc
+        (((s.whisker h).hom ≫ (precomp x f).map (i.right ▷ h)) ≫ (α_ f t.extension h).inv)
```

Comment:
Is there a reason this needs to be here? Specifically, is this change a stylistic one or necessary because of the new notation? It's generally much better to avoid stylistic changes in a PR which is making any other form of change.